.github/workflows/simulation.yml

```diff
@@ -320,6 +322,22 @@ jobs:
         ulimit -c unlimited
         ./script/test prepare_coredump_upload
         OT_OPTIONS='-DOT_READLINE=OFF -DOT_FULL_LOGS=ON -DOT_LOG_OUTPUT=PLATFORM_DEFINED' VIRTUAL_TIME=0 OT_NODE_TYPE=rcp ./script/test clean build expect
+    - name: Check Crash
+      if: ${{ failure() }}
+      run: |
+          CRASHED=$(./script/test check_crash | tail -1)
+          [[ $CRASHED -eq "1" ]] && echo "Crashed!" || echo "Not crashed."
+          echo "CRASHED=$CRASHED" >> $GITHUB_ENV
```

Comment:
Here we write this environment variable multiple times in one job. Did you verify the behavior in such situations?

Reply:
Good catch! I'll do some investigation on this.

Reply:
I used a simple solution for this: add a postfix for the variable in different runs in the same job. Then there won't be any conflicts. Thoughts?